Styles/portfolio.css

```diff
@@ -0,0 +1,77 @@
+body, htmls{
```

Comment:
I think you meant `html`:

```suggestion
body, html {
```